fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/lao/event/rollcall/RollCallFragment.kt

```diff
@@ -342,17 +345,30 @@ class RollCallFragment : AbstractEventFragment {
 
   companion object {
     val TAG: String = RollCallFragment::class.java.simpleName
+    private val deAnonymizationWarned = MutableLiveData(false)
```

Comment:
Why putting this as a static constant when the scope it's the Fragment instance?

The method isAttendeeListSorted is called from the RollCallAdapter, to which you can pass either the reference for such live data or the fragment itself.